pyshelf/resource_identity.py

```diff
@@ -8,11 +8,11 @@ def __init__(self, resource_url, bucket_name=None, artifact_path=None):
             resource_url = "/" + resource_url
 
         # To get rid of redundant separators
-        self.resource_url = os.path.normpath(resource_url)
+        self._resource_url = os.path.normpath(resource_url)
```

Comment:
different variables. resource_url and resource_path. It isn't used anywhere else so I added an underscore but now I just removed the assigning the url on the class level.